platform.md

```diff
@@ -148,6 +159,441 @@ A platform MAY support any number of mixins for a given stack in order to suppor
 Changes introduced by mixins SHOULD be restricted to the addition of operating system software packages that are regularly patched with strictly backwards-compatible security fixes.
 However, mixins MAY consist of any changes that follow the [Compatibility Guarantees](#compatibility-guarantees).
 
+## Lifecycle Interface
+### Platform API Compatibility
+
+The platform SHOULD set `CNB_PLATFORM_API=<platform API version>` in the lifecycle's execution environment
+
+IF `CNB_PLATFORM_API` is set in the lifecycle's execution environment, the lifecycle:
+  - MUST EITHER conform to the matching version of this specification OR
+  - Fail if it does not support `<platform API version>`
+
+### Operations
+
+#### Build
+A single app image build* consists of the following phases:
+
+1. Detection
+1. Analysis
+1. Cache Restoration
+1. Build*
+1. Export
+
+A platform MUST execute these phases EITHER by invoking the following phase-specific lifecycle binaries in order:
```

Comment:
I'd suggest only capitalizing the words defined in RFC 2119.

Reply:
Happy to make this change. I started doing this b/c the buildpack spec sometimes capitalizes non RFC 2119 words like `IF`, `THEN`, and `GIVEN`. Do you want me to change just the `EITHER` and `OR`s or all of the `IF`s as well?